src/radCommonAutoComplete.js

```diff
@@ -1,16 +1,12 @@
-import jQuery from 'jquery';
-
-require('jquery-ui');
-require('jquery-ui/ui/widgets/autocomplete');
-require('jquery-ui/ui/widgets/dialog');
+import $ from 'jquery';
```

Comment:
I'm sure this is fine, but a little curious how this works, I see the jquery.js is setting this on the global window object, which makes sense, but then we are importing here anyways?  Or maybe now that I'm looking at it this import is not referring to our "src/jquery.js" but instead the npm jquery, ok I think thats fine, was a little confused at first because of the naming